vnext/Shared/HermesRuntimeHolder.cpp

```diff
@@ -90,7 +93,11 @@ void HermesRuntimeHolder::initRuntime() noexcept {
   auto hermesRuntime = makeHermesRuntimeSystraced(runtimeConfig);
   facebook::hermes::HermesRuntime &hermesRuntimeRef = *hermesRuntime;
 
+#ifndef NDEBUG
```

Comment:
super nit: Consider avoiding double negative by just doing `#ifdef DEBUG`.

Reply:
I thought NDEBUG is a slight bit more standard .. And i'm confused about what is more reliable between DEBUG and _DEBUG !